.github/workflows/release.yml

```
@@ -38,37 +47,98 @@ jobs:
+          -t dashpay/quorum-list-server:${{ steps.meta.outputs.version }} \
+          $(for f in ${{ runner.temp }}/digests/*; do echo "dashpay/quorum-list-server@sha256:$(basename $f)"; done | xargs)
+
+    - name: Download binaries
+      uses: actions/download-artifact@v4
+      with:
+        pattern: binary-*
+        path: ${{ runner.temp }}/binaries
+        merge-multiple: true
+
     - name: Create GitHub Release
       uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)